rand_chacha/src/chacha.rs

```diff
@@ -19,6 +19,11 @@ use rand_core::{CryptoRng, Error, RngCore, SeedableRng};
 const STREAM_PARAM_NONCE: u32 = 1;
 const STREAM_PARAM_BLOCK: u32 = 0;
 
+// NB. this must remain consistent with some currently hard-coded numbers in this module
+const BUF_BLOCKS: u8 = 4;
+// number of 32-bit words per ChaCha block (fixed by algorithm definition)
+const BLOCK_WORDS: u8 = 16;
```

Comment:
These are used as `u64` / `u128` and should be inlined anyway, so use of `u8` here just adds noise.

Reply:
I was thinking this module might be further improved in the future to use values based on the named constants rather than hard-coded numbers for its `usize` values. `u8` is the only type that can be converted to both `u64` and `usize` with the total functions.